nltk/tokenize/moses.py

```diff
@@ -382,8 +383,9 @@ def tokenize(self, text, agressive_dash_splits=False, return_str=False):
         text = re.sub(regexp,substitution, text).strip()
         # Restore multidots.
         text = self.restore_multidots(text)
-        # Escape XML symbols.
-        text = self.escape_xml(text)
+        if not self.no_escaping:
```

Comment:
If we were to allow the function instead of the class object to take the escape parameter, then here we would just need:
```
if escape:
    text = self.escape_xml(text)
```